### ISSUE TITLE
Remote Server

### ISSUE THREAD

**Issue by KillahB33** [SchraderR/sp-tarkov-client#171]
Wondering what it would take for this to be able to manage a remote server?

I know migrating from electron to web app isn't as simple as removing the electron stuff but I was toying with doing that, but it seems like you have done an amazing job and wanted to reach out to you before I go forking and messing around with stuff.

**Comment by SchraderR** [comment]
@KillahB33 Thanks a lot! :) 

Manage remote server... so you got a remote root server with FIKA running and you want to use the tool to manage the mods without accessing the server?

Did i understand it correct?

**Comment by KillahB33** [comment]
Correct. I would prefer if I could use the tool to manage both the client and the remote server. I have a docker container currently running https://dev.sp-tarkov.com/Cbr/spt-docker. I use your tool to get all the mods, and then I copy the server side stuff manually to the server using VSCode. If I could skip the VS Code piece that would be amazing cause it's a bit cumbersome.
